### DISCUSSION TITLE
Cypress can not open Outlook screen (something went wrong) cette erreur apparait dans le console de cypress (uncaught exception)ChunkLoadError: Loading chunk 27969 failed. (error: https://res.public.onecdn.static.microsoft/owamail/hashed-v1/scripts/owa.27969.m.220e2fa0.js)

### DISCUSSION THREAD

**Discussion by Eyatester** [cypress-io/cypress#30152] [Questions and Help]
i want to login outlook to do some forget password tests , but Outlook appears always (something went wrongà and the Inbox does not appear this my code 
// Visitez l'URL initiale
cy.visit('https://outlook.office.com/');

// Gérez la connexion à l'origine Microsoft
cy.origin('https://login.microsoftonline.com', () => {
  cy.visit('/'); // Visitez la page de connexion

  // Remplissez les informations de connexion
  cy.get('#i0116').type('******@.com.tn');
  cy.get('#idSIButton9').click();
  cy.get('#i0118').type('*******');
  cy.get('#idSIButton9').click({ force: true });

  // Vérifiez et cochez la case "Se souvenir de moi"
  cy.get('#KmsiCheckboxField').click();

 

  // Cliquez sur le bouton de connexion si nécessaire
  cy.get('#idSIButton9').click({ force: true });
 
   // Attendez que l'URL soit celle attendue
  // cy.url().should('include', 'http://www.office.com/?auth=2');
});
Cypress.on('uncaught:exception', (err, runnable) => {
  // Ignorer uniquement les erreurs liées à des fonctionnalités expérimentales
  if (err.message.includes('experimental')) {
    return false;
  } else {
    // Logger l'erreur pour un débogage plus facile
    cy.log(`Erreur non gérée : ${err.message}`);
  }
});
Cypress.on('uncaught:exception', (err, runnable) => {
  return false; // Ignore les erreurs non critiques
})
// Attendez que la navigation vers l'URL correcte se produise
cy.origin('outlook.office.com', () => {
 
    // Assurez-vous de nettoyer les cookies et le stockage local pour éviter les interférences

  Cypress.on('uncaught:exception', (err, runnable) => {
  //   // returning false here prevents Cypress from
  //   // failing the test
  return false
 })

  
  // cy.clearLocalStorage()
  cy.visit("https://outlook.office.com")

  cy.clearCookies()
  cy.clearLocalStorage();

  
  cy.wait(10000)
  // Intercepter les requêtes réseau et vérifier les statuts
  cy.window().then((win) => {
    cy.stub(win, 'open').callsFake((url) => {
      cy.log(`Tentative d'ouverture de l'URL : ${url}`);
      cy.visit(url); // Forcer l'ouverture de l'URL dans le même onglet
    });
  });
  cy.reload();

  //cy.get('button').click()
  cy.wait(10000)
  // Attendez que l'URL correcte avec les paramètres de requête soit chargée
  // cy.url().should('include', '/?auth=2');
  // Vérifiez que le bouton de messagerie est visible et cliquez dessus
  //cy.get('#Mail > .fui-Button__icon').should('be.visible').click({ force: true });
  // cy.get('#Mail > .fui-Button__icon').click({ force: true })

});
![image](https://github.com/user-attachments/assets/38199440-fc7d-4f31-a716-3ffececdef9b)
this error always appear:  (uncaught exception)ChunkLoadError: Loading chunk 27969 failed. (error: https://res.public.onecdn.static.microsoft/owamail/hashed-v1/scripts/owa.27969.m.220e2fa0.js)

**Comment by portyu9** [comment]
Hi @Eyatester 

The important error here is:

```text
ChunkLoadError: Loading chunk 27969 failed
```

That means the **Outlook application itself failed to load one of its JavaScript chunks** from Microsoft's static CDN.

Returning `false` from `uncaught:exception` only tells Cypress not to fail the test because of the exception. It does **not** make the missing JavaScript chunk load, so Outlook can still remain in a broken state.

There are also two things in the test I would remove:

```js
cy.clearCookies()
cy.clearLocalStorage()
```

Doing that immediately around the Outlook login/navigation can destroy authentication/session state that Outlook needs.

I would also avoid multiple blanket handlers such as:

```js
Cypress.on('uncaught:exception', () => false)
```

because they can hide the actual application failure. Only suppress a specific, understood, non-fatal exception.

For the `ChunkLoadError` itself, inspect the failed `.js` request in the Network panel and determine whether it is returning `404`, being blocked by a proxy/security product, timing out, etc. The thread does not contain enough evidence to say which of those is happening.

For a **forgot-password** E2E flow, I would also avoid automating the Outlook inbox UI entirely. A much more stable approach is:

1. Submit the forgot-password form in your application.
2. Read the dedicated test mailbox programmatically, for example through Microsoft Graph.
3. Extract the reset link/token.
4. Visit your application's reset URL with Cypress.

That tests your password-reset workflow without making the test depend on Microsoft's changing Outlook web UI.

I could not retrieve the attached screenshot because its old GitHub attachment URL now returns 404, so the diagnosis above is based on the posted exception and code only.

Please mark this answer as accepted if it helped, thank you.
